wkcuber/api/Properties/LayerProperties.py

```diff
@@ -107,6 +115,7 @@ def _from_json(
                 else None,
             ),
             json_data["boundingBox"],
+            default_view_configuration=json_data.get("defaultViewConfiguration"),
```

Comment:
Unfortunately you also have to do this also for `SegmentationLayerProperties` (I don't know if there is a smarter way with inheritance)